meetings/2020-12-15.rst

```diff
@@ -0,0 +1,76 @@
+=========================
+Meeting December 15, 2020
+=========================
+
+- With James Hester present, the discussion naturally started on features of the CIF standard and thoughts on how our effort relates to the ontology-related work of the IUCr.
+  A context for this was a previous discussion on how an ontology for materials databases relates to / should be positioned to the work of IUCr and the crystallography domain of the EMMO.
+
+  * Discussion on the degree to which relationships are/can be represented in the CIF format (and thus the implicit ontology of that standard)
+  
+    - The CIF format is, at its core, "a dictionary" with carefully defined entities that can have values (single values, tables, etc.) 
+      
+    - The definitions of these data values describe relations between them, but there is no explicit machine-readable representation of relations.
```

Comment:
Very relevant change suggested by @jamesrhester with some edits by me to include more of the useful information from his comment:
```suggestion
    - The CIF standards consist of 2 syntaxes (CIF1 and CIF2), ontology languages (DDL2,DDLm and the older DDL1), and a series of what we call "dictionaries" which are ontologies written in DDLm or DDL2 for particular subject areas, e.g., "core", "modulated structures", "magnetic", "powder" and so forth. 

    - The dictionaries carefully define entities ("data names") that can have one or more values when appearing in data files. Sufficient information is provided in the dictionary to form a relational schema from the data names, where these data names are the columns appearing in the relations.

    - The definitions of the data names describe only those relationships found in the relational model: "key" data names and "foreign keys". The particular meaning underlying the foreign key relationship is not expressed.
```
Feel free to suggest further improvements; but if I don't hear anything more (over a few days) I'll commit this.

(Pinging @CasperWA.)

Reply:
All of the comments by @jamesrhester seem relevant, so great to see them added here 👍 Thanks @rartino and @jamesrhester.